package.json

```diff
@@ -2,11 +2,11 @@
   "name": "reactjs-popup",
   "version": "1.1.4",
   "description": "React Popup Component - Modals,Tooltips and Menus —  All in one",
-  "main": "reactjs-popup.cjs.js",
-  "module": "reactjs-popup.es.js",
-  "jsnext:main": "reactjs-popup.es.js",
-  "unpkg": "reactjs-popup.js",
-  "jsdelivr": "reactjs-popup.js",
+  "main": "lib/reactjs-popup.cjs.js",
```

Comment:
here, you shouldn't add the lib directory in entry file  because we publish the lib directory to NPM registry, not the whole project